scripts/seed.js

```diff
@@ -71,62 +71,58 @@ const seed = {
 }
 
 function getRandomColumnValue(columnName, tableName) {
-  let value = null
-
   switch (columnName) {
     case 'id':
-      value = undefined
-      break
+      return undefined
+    case 'asset_id':
+      return `${Math.floor(Math.random() * 307) - 153},${Math.floor(
```

Comment:
I think that `Math.floor(Math.random() * 307) - 153` might not be completely obvious if the reader doesn't know that asset_id represents.
How about `getRandomCoordinate()` ?